### ISSUE TITLE
[New Rule] aria-placeholder is valid on input type=email, number, password, search, tel, text, url (but not if placeholder is used)

### ISSUE THREAD

**Issue by carmacleod** [IBMa/equal-access#351]
Steps:
- go to https://w3c.github.io/html-aria/tests/placeholder-test.html
- run accessibility checker
- note violation flagged for "The WAI-ARIA role or attribute 'aria-placeholder' is not valid for the element `<input>`" on number & password
- all 7 input types that allow placeholder attribute need to allow aria-placeholder (including number & password)

**Comment by MHoov** [comment]
As of June 1
- [x] general rule description (SME & Dev)
- [x] testcases written (Dev)
- [ ] rule developed  (Dev)
- [ ] message written (SME)
- [ ] help written (SME)

**Comment by aliunwala** [comment]
From triage:
Looks like a new rule similar to epic #666 but might need a different message.

**Comment by shunguoy** [comment]
close as fixed per #800

**Comment by eirckr** [comment]
Validated 
<img width="1371" alt="Screen Shot 2022-05-03 at 11 51 19 AM" src="https://user-images.githubusercontent.com/62436670/166501326-ce8c4201-4569-4211-82eb-f4e09912dda9.png">
